api/constants/constants.go

```diff
@@ -294,6 +294,10 @@ const (
 	// DeviceTrustModeRequired enforces the presence of device extensions for
 	// sensitive endpoints.
 	DeviceTrustModeRequired DeviceTrustMode = "required"
+	// DeviceTrustModeRequiredHuman enforces the presence of device extensions
+	// for sensitive endpoints if the user is human. In this mode, bots are
+	// exempt from device trust checks.
+	DeviceTrustModeRequiredHuman DeviceTrustMode = "required-human"
```

Comment:
Definitely in nitpick territory and probably solvable with clear documentation, but `required-human` initially reads to me like it should _exclude_ bots (i.e. only humans can access sensitive endpoints, or what `required` actually does) rather than exempt them from checks?

`required-bots-exempt` doesn't have quite the same ring to it, though - maybe just something to make certain is clear when writing up the docs?

Reply:
Not a nitpick, that's really useful UX feedback! What about `required-for-humans`?